src/Xamarin.Android.Build.Tasks/Tasks/AndroidCreateDebugKey.cs

```diff
@@ -9,24 +9,21 @@ public class AndroidCreateDebugKey : KeyTool
 	{
 		public override string TaskPrefix => "ACD";
 
-		public int Validity { get; set; }
+		public int Validity { get; set; } = 30 * 365;
```

Comment:
We shouldn't lose the `// 30 years` comment here.  Otherwise we don't know what the unit is.